src/requirements/admin/migrate-engineering-liberal-arts-data.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+
+import { usernameCollection } from '../../firebase-admin-config';
+import { requirementAllowDoubleCounting } from '../requirement-frontend-utils';
+import getUserRequirementDataOnAdmin from './requirement-graph-admin-utils';
+
+const REQUIREMENT_ID = 'College-EN-Liberal Studies: 6 courses';
+
+async function runOnUser(userEmail: string) {
```

Comment:
Can you add documentation for this function

Reply:
Added